src/main.jl

```diff
@@ -13,6 +13,7 @@
         ignore::Int=adapt ? 1 : 0,
         measure::Union{Nothing,Function}=nothing,
         measurefreq::Int=1,
+        inplace::Bool=false,
         kwargs...
```

Comment:
Update function signature of `integrate` in docstring with new argument `parallel::Symbol=:nothread`

Reply:
Fixed!